backends/postgres/postgres_backend.go

```diff
@@ -31,21 +31,20 @@ import (
 var migrationsFS embed.FS
 
 const (
-	PendingJobIDQuery = `SELECT id
+	JobQuery = `SELECT id,fingerprint,queue,status,deadline,payload,retries,max_retries,run_after,ran_at,created_at,error
```

Comment:
The diff does a poor job of explaining this change. I've eliminated `PendingJobQuery` and replaced it with `JobQuery`, which has fewer filters, and should perform better.

Because of the setting in which `PendingJobQuery` was executed, it never needed to have the filter `run_after <= NOW()`. When that query is executed, the `jobID` is already pre-vetted as a pending job.